frontend/src/components/panels/project_filters_panel/filter_map_components/DateRangeSlider.js

```diff
@@ -1,9 +1,73 @@
 import React from "react";
+import ReactDOM from 'react-dom';
+import Tooltip from 'rc-tooltip';
+import Slider, { Range } from 'rc-slider';
+import 'rc-slider/assets/index.css';
+import 'rc-tooltip/assets/bootstrap.css';
 
-const DateRangeSlider = () => (
-    <div className="my-1">
-        <input type="range" className="custom-range" id="customRange1"/>
-    </div>
-);
+const createSliderWithTooltip = Slider.createSliderWithTooltip;
+const Handle = Slider.Handle;
+
+const dotStyle = {
+	'bottom': '-10px',
+    'width': '1px',
+    'border': '1px solid #e9e9e9',
+    'border-radius': '0%',
+    'margin-left': '-1px',
+};
+
+const handle = (props) => {
+  const { value, dragging, index, key, className, disabled, offset, prefixCls} = props;
+  return (
+    <Tooltip
+      prefixCls="rc-slider-tooltip"
+      overlay={value}
+      visible={dragging}
+      placement="top"
+      key={index}
+    >
+      <Handle value={value} dragging={dragging} index={index} key={key} className={className} disabled={disabled} offset={offset} prefixCls={prefixCls} />
```

Comment:
React error: `Warning: Received "false" for a non-boolean attribute "dragging".`

`dragging` doesn't appear to be a prop for the `Handle` component. If so, it should accept a string value.